extensions/ql-vscode/src/helpers.ts

```diff
@@ -29,8 +29,14 @@ export async function showAndLogErrorMessage(message: string, {
   items = [] as string[],
   fullMessage = undefined as (string | undefined)
 } = {}): Promise<string | undefined> {
-  return internalShowAndLog(message, items, outputLogger, Window.showErrorMessage, fullMessage);
+  const NUM_LINES_TO_KEEP = 2;
+  return internalShowAndLog(dropLinesExceptInitial(message, NUM_LINES_TO_KEEP), items, outputLogger, Window.showErrorMessage, fullMessage);
```

Comment:
See other suggestion below.

```suggestion
  return internalShowAndLog(dropLinesExceptInitial(message), items, outputLogger, Window.showErrorMessage, fullMessage);
```